src/content/docs/plugin/file-system.mdx

```diff
@@ -84,6 +84,14 @@ Install the fs plugin to get started.
 
 The fs plugin is available in both JavaScript and Rust.
 
+:::note[Rust Exposure]
+
+The exposed API on the Rust side is mostly intended to manipulate the [Scope](#scopes) of the plugin.
+It does expose some APIs to access the file system on mobile, but the recommendation is to use [`tokio::fs`](https://docs.rs/tokio/latest/tokio/fs/index.html) or the
```

Comment:
For Android the tokio or std APIs won't work if you're using e.g. the dialog API (which returns a content:// URI instead of a path to the file). We might also need more changes for iOS to properly request for permissions to access some files (for instance photos, or the Files location).
I think in the end when the FS plugin is marked as stable, we'll recommend using it even for Rust code unless your app is desktop-only.

Reply:
:flip-table: 😂 
Alright, so we recommend std/tokio for desktop and our api for mobile?

Reply:
we'll fallback to std for desktop, so if you need cross platform support for e.g. reading resources and dialog-selected files, you need to use our own API wrapper